device/bgp/bgpsrtepolicynamesubtlv.yaml

```diff
@@ -0,0 +1,16 @@
+components:
+  schemas:
+    BgpSrte.policyNameSubTlv:
+      description: >-
+        Configuration for the Policy Name sub-TLV.
+        The Policy Name sub-TLV is attach a symbolic name
```

Comment:
The Policy Name sub-TLV is used to attach ...

Reply:
done.